packages/hoppscotch-common/src/components/app/PaneLayout.vue

```diff
@@ -30,25 +30,40 @@
       </Splitpanes>
     </Pane>
     <Pane
-      v-if="SIDEBAR && hasSidebar"
+      v-if="mdAndLarger && SIDEBAR && hasSidebar"
       :size="PANE_SIDEBAR_SIZE"
       min-size="20"
       class="flex flex-col !overflow-auto bg-primaryContrast"
     >
       <slot name="sidebar" />
     </Pane>
+
+    <!-- Mobile Collections Layout -->
+    <SmartSlideOver
+      v-if="!mdAndLarger && SIDEBAR && hasSidebar"
+      :show="show"
+      :title="t('tab.collections')"
+      @close="SIDEBAR = !SIDEBAR"
+    >
+      <template #content>
+        <div class="!overflow-auto">
+          <slot name="sidebar" />
+        </div>
+      </template>
```

Comment:
Change the name #content to something meaningful as slideovercontent

Reply:
Renamed #content to #slideovercontent